cmd/createdb.go

```
@@ -28,9 +27,14 @@ var createDBCmd = &cobra.Command{
         if userName == "" {
             userName = serviceName
         }
-        err := cfgMgr.CreateDB(serviceName, userName)
+        dbPassword := cfgMgr.GetRandomPassword()
+        dbCreated, err := cfgMgr.CreateDB(serviceName, userName)
```

This error is not being checked